app/views/users/folders/binaries/show.html.erb

```diff
@@ -5,4 +5,11 @@
 <%= render_preview(@binary) %>
 </div>
 
-<%= render partial: 'shared/comments' %>
\ No newline at end of file
+<%= render partial: 'shared/comments' %>
```

Comment:
also, neither the form nor the display is styled yet - i can look at what we have done so far and update that